R/Design-class.R

```diff
@@ -887,6 +887,11 @@ DADesign <- function(model, data,
 #'   concurrent mono and combo cohorts.
 #' @slot same_dose (`flag`)\cr whether the lower dose of the separately determined mono and combo
 #'   doses should be used as the next dose for both mono and combo.
+#' @slot parallel_start (`flag`)\cr whether for the time when mono and combo are the first time together,
+#'   the same dose should be used. Note that this is different from `same_dose` which will always force
+#'   them to be the same. This is useful as an option when we want to use `first_cohort_mono_only = TRUE`
```

Comment:
The description uses slightly awkward phrasing.  Perhaps "indicates whether, when mono and combo cohorts run in parallel for the first time, the same dose should be used. ..."